### PR TITLE
Allow bypassing deployment validation and expose basic subroutine name on connection instance

### PR DESCRIPTION
**Pull request by shawnmcknight**
This PR makes the following changes:

#### Allow bypassing deployment validation
A new `validateDeployment` property has been added to the `open` method options on the `Connection` class. This option, which defaults to `true`, indicates that the database server's required mvom subroutine will be validated prior to opening the connection. This validation consists of checks to ensure that the MVIS subroutine definition exists and that the subroutine has been cataloged on the server.

If this option is overridden to `false`, this check will be bypassed and the connection will attempt to be opened.

#### Expose multivalue subroutine name on `Connection`
A new public property has been added to the `Connection` object named `subroutineName`. This is a public getter which will return the name of the subroutine as determined by the `DeploymentManager` instance associated with the `Connection`. This property can be accessed by consumers to determine what the name of the subroutine used by mvom when communicating through MVIS to the database server is.

### CODE REVIEW

**Review by philfuster — CHANGES_REQUESTED** [review]
This looks good to me. I just noticed a comment that I think could be removed.

**Review by kthompson23 — APPROVED** [review]
Looks good, thank you.

File: src/Connection.ts [hunk 1]
```diff
@@ -101,7 +101,13 @@ interface RequestOptions {
 	requestId?: string;
 }
 
-export type OpenOptions = RequestOptions;
+export interface OpenOptions extends RequestOptions {
```
[the remark this answers is not in the document]
Comment (shawnmcknight) [reply src/Connection.ts:103]: Removed

**Review by philfuster — APPROVED** [review]
LGTM! Thank you.

**Pull request merged**